Makefile

```diff
@@ -1,16 +1,42 @@
-GFA_FILES := k.gfa DRB1-3123.gfa
+TEST_FILES := t k note5 overlap q.chop DRB1-3123 chr6.C4 LPA
+BASIC_TESTS := ex1 ex2
 GFA_URL := https://raw.githubusercontent.com/pangenome/odgi/ebc493f2622f49f1e67c63c1935d68967cd16d85/test
+GFA_ZIP_URL := https://s3-us-west-2.amazonaws.com/human-pangenomics/pangenomes/scratch/2021_05_06_pggb/gfas/chr8.pan.gfa.gz
 
-.PHONY: fetch test clean
-fetch: $(GFA_FILES)
+.PHONY: fetch og test clean test-all
+fetch: $(TEST_FILES)
 
-test: $(GFA_FILES)
-	turnt test/$^
+og: fetch $(foreach file,$(TEST_FILES),test/$(file).og) $(foreach file,$(BASIC_TESTS),test/basic/$(file).og)
 
-clean: 
-	rm -rf test/*.gfa
+test: og
+	-turnt --save --env baseline $(foreach file,$(TEST_FILES),test/$(file).gfa)
```

Comment:
My intuition was that it is more in keeping with the philosophy of differential testing to generate the `.out` files every time we run the tests, especially since we get some of our tests from [the odgi repo](https://github.com/pangenome/odgi/tree/e2de6cbca0169b0720dca0c668743399305e92bd/test), and these tests could theoretically change without us knowing about it.

Reply:
Sounds great to me.